packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -215,6 +314,9 @@ async function abortBuild() {
 
         <div hidden="{buildImageInfo?.buildRunning}">
           <label for="containerBuildPlatform" class="block mb-2 text-sm font-bold text-gray-400">Platform</label>
+          {#if platforms.length > 1}
+            <p class="text-sm text-gray-600 mb-2">Multiple platforms selected, a manifest will be created</p>
```

Comment:
I feel like a new user would not know what is a manifest here. Maybe we could add a `fa-circle-info` with a tooltip, giving a very short explanation on what is manifest